scripts/main.js

```diff
@@ -0,0 +1,45 @@
+// spotify functions
+spotifyURL = 'https://api.spotify.com/v1/';
```

Comment:
I think JS accepts this, but generally good practice to declare it as a `const` here